frontend/src/components/mine/ContactInfo/PartyRelationships/DefaultContact.js

```diff
@@ -118,18 +118,21 @@ export class DefaultContact extends Component {
             </Button>{" "}
             {this.props.isEditable && [
               <br />,
-              <ConditionalButton
-                type="primary"
-                string="Update"
-                handleAction={() =>
-                  this.props.openEditPartyRelationshipModal(
-                    this.props.partyRelationship,
-                    this.props.onSubmitEditPartyRelationship,
-                    this.props.handleChange,
-                    this.props.mine
-                  )
-                }
-              />,
+              <AuthorizationWrapper permission="role_create">
+                <Button
+                  type="primary"
+                  onClick={() =>
+                    this.props.openEditPartyRelationshipModal(
```

Comment:
This is quite a few positional arguments. If this function gets any more arguments (or if any become/are already optional), we may want to refactor this to accept an object, for the benefit of having named arguments.

Not necessarily related to this PR, but I'd like to flag it

Reply:
Good point, Should bring this to the attention of @nayr974

Reply:
@NWCalvank My thought is that as this function is defined and consumed only within this component, and that all of our open modal functions in the app use only positional arguments that leaving it is better for consistency. I see a better argument for refactoring "Contact" proptypes into an object as those are passed through a couple layers. I'll keep both of these in mind if we are working on this screen again.